rosidl_parser/rosidl_parser/parser.py

```diff
@@ -103,16 +103,19 @@ def extract_content_from_ast(tree):
     constants = {}
     const_dcls = tree.find_data('const_dcl')
     for const_dcl in const_dcls:
+        annotations = get_annotations(const_dcl)
         const_type = next(const_dcl.find_data('const_type'))
-        const_expr = next(const_dcl.find_data('const_expr'))
+        *_, const_expr_value = const_dcl.find_data('const_expr')
```

Comment:
@ivanpauno nit: isn't this still a `const_expr`:
```suggestion
        *_, const_expr = const_dcl.find_data('const_expr')
```
?

Reply:
Yes, it's the `const_expr` representing the value of the constant.
The annotations can also have some `const_expr` in the middle, so this was the way to get the one corresponding to the value.

Reply:
Actually, I think that I should only be checking childs like [here](https://github.com/ros2/rosidl/pull/593#discussion_r630455973).
@hidmic what do you think?

Reply:
> What you're getting is, strictly speaking, a const_expr object / thingie. That's why I pointed it out. It's just nit though,

Ah I see, I didn't notice that `const_expr_value` was also a thing.

> I don't have a strong opinion. Checking direct children is probably cheaper than a full scan though.

Simplified that in https://github.com/ros2/rosidl/pull/593/commits/16d079203c5f9c7d2877cb5b914407129dff4fde.